tests/org.eclipse.n4js.xpect.tests/xt-tests/dts-parser/user-tests/misc/noDuplicateImportErrorDueToOverloading.n4js.xt

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2022 NumberFour AG.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   NumberFour AG - Initial API and implementation
+ */
+
+/* XPECT_SETUP org.eclipse.n4js.xpect.tests.N4jsXtTest
+
+	File "other.d.ts" {
+
+		declare module "a/b/someModule" {
+			export function foo(p: string): number;
+			export function foo(p1: string, p2: number): number;
```

Comment:
If you merged master, then this test would that the DtsBuilder collapses overloaded functions (see ParserContextUtils#removeOverloadingFunctionDefs()).

... which still is a valid test case!

Reply:
I've just double checked this:
1. as you say, the above test is green even without the "localNamesAdded" set in `ImportSpecifiersUtil`, thanks to your changes from master.
2. I'm adding a similar second test using a variable and a namespace of same name that will still require the "localNamesAdded" set.